tests/test_api.py

```diff
@@ -1457,3 +1457,13 @@ def test_srpm_component_provides_sources_upstreams(client, api_path):
     for provide in response.json()["sources"]:
         assert "name" in provide
         assert "purl" in provide
+
+
+@pytest.mark.django_db(databases=("default", "read_only"), transaction=True)
+def test_product_streams_exclude_components(client, api_path):
+    stream = ProductStreamFactory(exclude_components=["golang"])
+
+    response = client.get(f"{api_path}/product_streams?name={stream.name}")
+    assert response.status_code == 200
+    assert len(response.json()["exclude_components"]) == 1
+    assert response.json()["exclude_components"] == ["golang"]
```

Comment:
Let's do `response = response.json()` once and save the result. We also don't need to assert the len() of the value if we're asserting the exact contents of the value:
```suggestion
    assert response.status_code == 200
    # It's easier to understand any traceback / errors if the response.json() conversion is on its own line
    response = response.json()
    assert response["exclude_components"] == ["golang"]
```

Reply:
will do